### DISCUSSION TITLE
Slider whose value can be changed

### DISCUSSION THREAD

**Discussion by petrasvestartas** [ocornut/imgui#6087] [New Users Build/Link/Run issues ONLY!]
Is there a possibility to create a slider whose interval could be changed during render time?
For example the initial values are from 0.0 to 1.0, but the user would like to change it from -1 to 11.1.

Is there such an in-built method?

**Comment by PathogenDavid** [comment]
It's not totally clear exactly what you're asking for. A doodle or an example of where you've seen the widget you have in mind somewhere else would help a lot.

Just about every widget in Dear ImGui is featured in the official demo, you should look through it to see if something meets your needs. (Although based on your description nothing comes to mind.)

(As an aside, the [discussions category should not be used for this sort of question](https://github.com/ocornut/imgui/discussions/4497).)

**Comment by ocornut** [comment]
I seemingly can understand the question but simultaneously I don't understand your issue.
**The internal values are parameters to the function, you can feel to change them at runtime anyway you like.**
You can e.g. add a context menu to let the user change intervals, but you'll need to store that data.

Note that `DragFloat()` widgets don't need to have limits, unlike `SliderFloat()`.
